### PR TITLE
Oppilams

### PR DESCRIPTION
**Pull request by Oppila-ms**

### CODE REVIEW

**Comment by jia200x** [comment]
hi @Oppila-ms, welcome to the RIOT community!

In order to ease the review process, please follow the procedure described in [CONTRIBUTING.md](https://github.com/RIOT-OS/RIOT/blob/master/CONTRIBUTING.md) (Pull Request). 

The contribution is not clear to me (from what I can see it's a new board support), but there's missing information regarding how to test, which features are added, etc.

**Comment by Oppila-ms** [comment]
file checked

**Comment by benpicco** [comment]
This appears to be a duplicate of #14552

**Comment by jia200x** [comment]
and #14633

**Comment by Oppila-ms** [comment]
Hello 

Where to add testing procedures and yes it is duplicate. We are trying to port our omote board into riot-os since we are new to this area we are not aware some methods so trying hard in different ways to port.could you please help us port our board into riot. I just want to know what is the next step that i should take to port our board successfully.

Regards
Manikandan M

**Comment by Oppila-ms** [comment]
Hi
One of the riot member helped us to correct our file and we commited and pushed it agian after doing correction but after that we failed to create pull request. It would be more helpful if any of you help us to lead further.

Regards
Manikandan M

**Comment by jia200x** [comment]
> Hi
One of the riot member helped us to correct our file and we commited and pushed it agian after doing correction but after that we failed to create pull request. It would be more helpful if any of you help us to lead further.

Hi again,

There should be only one Pull Request where all requested changes should be pushed. Either this one, #14552 or #14633. So as a first step, please choose one and close the others, so maintainers can focus in only one Pull Request.

Then, please update at least the "Contribution description" and "Testing procedures" with the following information:

- What kind of board is it, which architecture, where to buy it, etc
- Features introduced in the PR
- State of the PR (is it WIP, ready to be merged? draft?)
- How to test this PR

Otherwise is not so easy for other maintainers to do a proper review.

> could you please help us port our board into riot.  I just want to know what is the next step that i should take to port our board successfully.


Of course! We are there to help.
Regarding the next steps, it usually goes like the following:

1. After the contribution description is updated (as requested above), one or more maintainers can review the code and possibly request changes
2. When changes are requested, all fixes should be pushed back to the branch.
3. When all changes are addressed (and test passes), it's ready to be merged by a maintainer.

If you have any questions on the process, don't hesitate to ask again here.

> One of the riot member helped us to correct our file and we commited and pushed it agian after doing correction but after that we failed to create pull request. It would be more helpful if any of you help us to lead further.

As explained above, there should be only one PR where to push changes to. Not one PR per change.

In a nutshell, please:
- Close the duplicate PRs
- Update this PR description (click the three dots in the upper right corner of the PR description box, and then Edit)
- Give more details about the problem you have regarding pushing changes to a PR

With this information we can focus our efforts in order to get the PR in shape.

**Comment by Oppila-ms** [comment]
Hello


We have updated with contribution and testing procedures. I would like to know next procedures.

Regards
Manikandan M
